proto/self_serve/orb/orb.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package self_serve.orb;
+
+enum OrbMode {
```

Comment:
`OrbMode` tells the app which version of communication (which messages) should it use to comunicate to the Orb it's communicating at the moment